src/pages/home/home.ts

```diff
@@ -200,9 +206,12 @@ export class HomePage {
 
     // This event is sent to an existent instance of Copay (only for standalone apps)
     gui.App.on('open', (pathData) => {
-      if (pathData.indexOf(/^bitcoin(cash)?:/) != -1) {
+      if (pathData.indexOf('bitcoincash:/') != -1) {
```

Comment:
Can we add some unit tests here? This is really important logic, and we need to make sure it doesn't ever break

Reply:
Hi @bitjson , Could you check this PR? We need to merge it to be able to send desktop. The home page tests are already being done in a separate PR.